homeassistant/components/prometheus/__init__.py

```diff
@@ -394,6 +399,34 @@ def _handle_device_tracker(self, state):
         value = self.state_as_number(state)
         metric.labels(**self._labels(state)).set(value)
 
+        # record battery level of the device tracker if available
+        self._battery(state)
+
+        # device-tracker entities have certain well known attributes that we can record here in prometheus as a
+        # time series.
+        well_known_attributes = {
+            ATTR_LATITUDE: (
+                "device_tracker_latitude",
+                "Latitude of the device tracker measured in decimal degrees",
+            ),
+            ATTR_LONGITUDE: (
+                "device_tracker_longitude",
+                "Longitude of the device tracker measured in decimal degrees",
+            ),
+            ATTR_GPS_ACCURACY: (
+                "device_tracker_gps_accuracy",
```

Comment:
What unit is this in? Prometheus [metric naming guidelines](https://prometheus.io/docs/practices/naming/) recommends having a suffix describing the unit, it would really help here.

Reply:
There is no official unit from what I can see, most of the devices I've seen will report meters but I don't think we can know whether unit conversion has already happened or not and whether the HA units configuration has taken effect. For that reason, I was leaving it off here.